haystack/text_propagator.py

```diff
@@ -0,0 +1,41 @@
+from .propagator import Propagator
+from .span import SpanContext
+from opentracing import SpanContextCorruptedException
+
+TRACE_ID = "Trace-ID"
```

Comment:
Can we keep it in a separate file where all constants throughout the project can be defined?

Reply:
refactored to constants.py